src/App.js

```diff
@@ -1,25 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import styled from 'styled-components'
+import { useState } from 'react'
+import Sightseeingpage from './pages/Sightseeingpage/SightseeingPage'
+import RestaurantPage from './pages/RestaurantPage/RestaurantPage'
+
+export default function App() {
+  const [currentPage, setCurrentPage] = useState('sightseeingPage')
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <AppGrid>
+      {currentPage === 'sightseeingPage' && (
+        <Sightseeingpage onNavigate={showSightseeingPage} />
+      )}
+
+      {currentPage === 'restaurantPage' && (
+        <RestaurantPage onNavigate={showRestaurantPage} />
+      )}
+    </AppGrid>
+  )
+
+  function showRestaurantPage() {
+    setCurrentPage('sightseeingPage')
+  }
+  function showSightseeingPage() {
+    setCurrentPage('restaurantPage')
+  }
```

Comment:
you could have just one function where you pass in the page name or even get rid of both functions because the only thing the functions do is `setCurrentPage()`which you could do directly